website/users.json

```diff
@@ -120,7 +120,7 @@
     "image": "/images/eventstag-200x100.png",
     "infoLink": "https://eventstag.com"
   },
-  {
+  
```

Comment:
I think you deleted that accidentally

Reply:
sorry for the error, the file is updated now, thanks